### DISCUSSION TITLE
【求助】寻布启事

### DISCUSSION THREAD

**Discussion by ShangHuaWenYuan** [share121/inter-knot#915] [其他]
本人在六分街散步时不小心弄丢一只全新艾利都最可爱的邦布，外貌如图，如果有好心人发现希望立即联系本人并将邦布送往白祇重工，必有重谢！
![20240810184517 - 副本](https://github.com/user-attachments/assets/6225f13c-470c-4308-8f19-43c2950c1ddf)

**Comment by SixRadium** [comment]
法厄同去白祇打工了？

**Comment by Mayf1ower** [comment]
是格莉丝吧！我说了我是不会再让你碰伊埃斯的！

**Comment by WLMRMZ** [comment]
这个我知道，在空洞里找黄金邦布去了，要不你去找找绳匠法厄同？

**Comment by warning-Kirov-reporting** [comment]
![image](https://github.com/user-attachments/assets/72fd4281-4be7-4d17-9822-38ace5437c9e)
嗯呐嗯呐，（在我这）嗯呐嗯呐！（兄弟们一起去抓黄金邦布了！）

**Comment by ArcNaive** [comment]
这个委托我接了！

**Comment by Larksonguesinaspic** [comment]
你兄妹俩去给白祇重工打工了？？？？？？？？？？
![image](https://github.com/user-attachments/assets/451891b8-4484-4570-852f-3b4a4b56ca52)

**Comment by Q99966** [comment]
好机会
